Include/internal/pycore_optimizer.h

```diff
@@ -33,6 +33,8 @@ struct _Py_UopsSymbol {
     int flags;  // 0 bits: Top; 2 or more bits: Bottom
     PyTypeObject *typ;  // Borrowed reference
     PyObject *const_val;  // Owned reference (!)
+    int tuple_count;
+    struct _Py_UopsSymbol **tuple_val;
```

Comment:
Can you use indexes, rather than pointers? We can fix 6 10-bit indexes into a 64 bit int.
We really don't care about tuples with `len() > 6` and we are already using too much stack memory for these data structures.

Would it be possible to use a union, and overlap this field with `const_val`?